src/qibolab/serialize.py

```diff
@@ -68,15 +68,22 @@ def load_qubits(
             for c, char in runcard["characterization"]["coupler"].items()
         }
 
-        for c, pair in runcard["topology"].items():
-            q0, q1 = pair
+        for pair, char in zip(
```

Comment:
```suggestion
        for (c, pair), char in zip(
```
is a bit more readable. As it is now, it is a bit hard to understand that `pair` contains both the coupler and the pair.